SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -3,32 +3,36 @@
 #pragma once
 
 #include <WorkerSDK/improbable/c_worker.h>
+
 #include "CoreMinimal.h"
 
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialLoadBalanceEnforcer, Log, All)
 
 class SpatialVirtualWorkerTranslator;
-class USpatialSender;
 class USpatialStaticComponentView;
 
 class SpatialLoadBalanceEnforcer
 {
 public:
-	SpatialLoadBalanceEnforcer();
+	struct AclWriteAuthorityRequest
+	{
+		Worker_EntityId EntityId = 0;
```

Comment:
For a struct I don't think this is really necessary.

Reply:
You can zero init an instance if you need to. Otherwise you plan on setting every value anyway.

Reply:
It isn't really necessary, but IMO being explicit is nice.